PrestaShop/PrestaShop
# Migrate Merchandise Return - Order Detail List

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | This PR migrates "merchandise return" product list to Symfony
| Type?         | refacto
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | https://github.com/PrestaShop/PrestaShop/issues/24421
| How to test?  | See below

## How to test

1. Enable merchandise returns option
2. Create some merchandise returns.
3. Enable the Merchandise return page in experimental features
4. Go to `admin-dev/index.php/sell/customer-service/merchandise-return/` which is the list. Click on any of the existing returns to open the page.
5. You should be able to see list of products in that merchandise return. They should be deletable using delete action or bulk action. Merchandise return must always have at least one product left.
